packages/web-app/src/components/appli/Entry/Snapshots/index.jsx

```diff
@@ -0,0 +1,66 @@
+import { useParams } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchSnapshot } from '../../../../actions/Snapshot/GetSnapshots';
+import { useEffect } from 'react';
+import * as React from 'react';
+import {
+  Accordion,
+  AccordionSummary,
+  AccordionDetails,
+  Typography
+} from '@material-ui/core';
+import ExpandMore from '@material-ui/icons/ExpandMore';
+import { useIntl } from 'react-intl';
+import ScrollableContent from '../../../common/Layouts/Fixed/ScrollableContent';
+import Translate from '../../../common/Translate';
+import Alert from '../../../common/Alert';
+
+const SnapshotPage = () => {
+  const { formatMessage } = useIntl();
+  const { id, type } = useParams();
+  const dispatch = useDispatch();
+  const data = useSelector(state => state.getSnapshots.data);
+
+  useEffect(() => {
+    dispatch(fetchSnapshot(id, type));
+  }, [id, dispatch]);
+
+  return (
+    <ScrollableContent
+      dense
+      title={formatMessage({ id: `Snapshots ${type}` })}
```

Comment:
`${type} snapshots`

Reply:
You must add this translation to the `en.json` file.

Reply:
Should we add only `snapshots` or `${type} snapshots` to en.json

Reply:
"Snapshots ${type}": "Snapshots ${type}" in en.json. Browse the en.json file: you will find some example of translated strings with a dynamic value.